src/Util/ValueHelper.php

```diff
@@ -9,8 +9,8 @@
 class ValueHelper
 {
     /**
-     * @param array $argumentsA
-     * @param array $argumentsB
+     * @param ArgumentNode[] $argumentsA
+     * @param ArgumentNode[] $argumentsB
      * @return bool
      */
     public function compareArguments(array $argumentsA, array $argumentsB): bool
```

Comment:
Can we make those static?

Reply:
The class is made via the container IIRC, but I guess we can change it